examples/with-apollo-graphql/src/client.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { ApolloProvider } from 'react-apollo';
+import { ApolloClient } from 'apollo-client';
+import { createHttpLink } from 'apollo-link-http';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+
+import App from './App';
+
+// Create an ApolloClient instance
+// Pass your GraphQL endpoint to uri
+const client = new ApolloClient({
+  link: createHttpLink({
+    uri: 'https://gql-placeholder.herokuapp.com/graphql',
```

Comment:
This URL doesn't appear to be valid any longer when directly accessed. The following URL does work though: http://gql-placeholder.herokuapp.com/playground.